test/check-machines-filesystems

```diff
@@ -114,6 +114,7 @@ class TestMachinesFilesystems(VirtualMachinesCase):
 
         # Start VM and ensure that adding filesystem is disabled
         b.click("#vm-subVmTest1-run")
+        b.wait_in_text("#vm-subVmTest1-state", "Running")
```

Comment:
That is not really needed. The next line (disabled button) is an implicit check for this, as the add button is disabled for running VMs.

This is failing on RHEL-9 https://bugzilla.redhat.com/show_bug.cgi?id=1966580 so if you decide to keep it you need to adjust the naughty

Reply:
Oh, heh, thanks -- I was about to file that bug, I thought it was a recent regression. Adding this seems to make the error more clear (as it then fails on a plausible thing, not "huh, did we break the disabling of the button?"). But as that already exists, I just back it out.